controllers/front/PaymentCard16.php

```diff
@@ -54,16 +54,19 @@ public function initContent()
 
         $paypalAccountRepository = new PaypalAccountRepository();
 
-        if (false === $paypalAccountRepository->cardPaymentMethodIsValid()) {
+        if (false === $paypalAccountRepository->cardHostedFieldsIsAvailable()) {
             $this->redirectToHomePage();
         }
 
         $paypalOrder = new CreatePaypalOrderHandler($this->context);
         $paypalOrder = $paypalOrder->handle();
 
         $language = (new LanguageAdapter())->getLanguage($this->context->language->id);
+        $paypalSdkLink = new PrestaShop\Module\PrestashopCheckout\Builder\PayPalSdkLink\PayPalSdkLinkBuilder();
```

Comment:
We are not in the main class of the module. It will be better to import the class using the use statement.